spec/models/user_spec.rb

```diff
@@ -12,4 +12,13 @@
   describe 'associations' do
     it { should have_many(:targets) }
   end
+
+  describe '#chats' do
+    let(:user_a) { create(:user) }
```

Comment:
I think you can just call it `user` instead of `user_a` . Because then you use it as both `user_a` and `user_b`